models/migrations/v1_24/v320.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2025 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package v1_24 //nolint
+
+import (
+	"code.gitea.io/gitea/modules/json"
+
+	"xorm.io/xorm"
+)
+
+func MigrateSkipTwoFactor(x *xorm.Engine) error {
+	type LoginSource struct {
+		TwoFactorPolicy string `xorm:"two_factor_policy NOT NULL DEFAULT ''"`
+	}
+	err := x.Sync(new(LoginSource))
```

Comment:
Use `SyncWithOptions` to avoid `index` sync.

Reply:
I don't understand the details. Feel free to edit directly and document it

Reply:
2a1d34a